### PR TITLE
processors now actually accepts monkeys

### PR DESCRIPTION
**Pull request by xTrainx**
:cl: xTrainx
fix: processors now actually accepts monkeys
/:cl:

[why]:  so fun fact, my last pr did actually turned dead monkeys into bloody buckets, but turns out i forgot to add a way for it to actually eat the monkeys, crazy right? tested locally, worked groovely, although, it now adds gibs along the bucket

### CODE REVIEW

**Comment by xTrainx** [comment]
look at that shit, man, first time travis accepts me at the first hit

**Comment by yoyobatty** [comment]
Is this for the slime processor or kitchen processor?

**Comment by yoyobatty** [comment]
This works too right?

**Comment by xTrainx** [comment]
the slime processor, kitchen processor is unaffected, and yes, it works, the only change from how it was before is that it also leaves some gibs around

**Comment by Lxtoofonea** [comment]
+1 ahhhh

**Review by yoyobatty — COMMENTED** [review]
And you've tested it too right?

**Comment by xTrainx** [comment]
Yes  did, it takes slimes normally and process it the dame, cant take humans and live monkeys are just ejected

File: hippiestation/code/modules/food_and_drinks/kitchen_machinery/processor.dm [hunk 1]
```diff
@@ -30,3 +30,52 @@
...
+	if(user.a_intent == INTENT_GRAB && ismob(user.pulling) && select_recipe(user.pulling))
+		if(user.grab_state < GRAB_AGGRESSIVE)
+			to_chat(user, "<span class='warning'>You need a better grip to do that!</span>")
+			return
+		var/mob/living/pushed_mob = user.pulling
+		user.visible_message("[user] put [pushed_mob] into [src].", \
+			"You put [pushed_mob] into [src].")
+		pushed_mob.forceMove(src)
+		user.stop_pulling()
+		return 1
```
Comment (yoyobatty) [inline hippiestation/code/modules/food_and_drinks/kitchen_machinery/processor.dm]: `return TRUE`

File: hippiestation/code/modules/food_and_drinks/kitchen_machinery/processor.dm [hunk 2]
```diff
@@ -30,3 +30,52 @@
...
+	else if (ismob(user.pulling))
+		if(user.a_intent != INTENT_HARM)
+			to_chat(user, "<span class='warning'>That probably won't blend!</span>")
+			return 1
```
Comment (yoyobatty) [inline hippiestation/code/modules/food_and_drinks/kitchen_machinery/processor.dm]: `return TRUE`

**Review by yoyobatty — CHANGES_REQUESTED** [review]

**Review by yoyobatty — APPROVED** [review]

**Pull request merged**